lib/ui/pages/news_list/primary_news_list_item.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:allthenews/ui/pages/news_list/primary_news_list_entity.dart';
+import 'package:flutter/material.dart';
+
+abstract class _Constants {
+  static const double newsImageRadius = 15;
+  static const double newsImageLayerOpacity = 0.7;
+  static const double authorImageRadius = 15;
+  static const double authorRightPadding = 8;
+  static const double stackItemPadding = 20;
+}
+
+class PrimaryNewsListItem extends StatelessWidget {
+  final PrimaryNewsListEntity news;
+
+  PrimaryNewsListItem({Key key, @required this.news})
+      : assert(news != null),
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Stack(
+        children: <Widget>[
+          ClipRRect(
+            borderRadius: BorderRadius.circular(_Constants.newsImageRadius),
+            child: Container(
+              color: Colors.black,
+              child: Opacity(
+                opacity: _Constants.newsImageLayerOpacity,
+                child: Image.network(
+                  news.imageUrl,
+                  //FIXME enter correct dimensions when filling a list with those items
+                  width: 300,
+                  height: 300,
+                  fit: BoxFit.fill,
+                ),
+              ),
+            ),
+          ),
+          Positioned(
+            top: 1,
+            right: 1,
+            child: _buildBookmarkIcon(),
+          ),
+          _buildNewsInfo(context),
```

Comment:
powiedzmy jako, że Stack bezpośrednio w metodzie build to proponowałbym wynieść Positioned z metody _buildNewsInfo do build, tak jak jest to zrobione w przypadku _buildBookmarkIcon - wtedy od razu widać jak wszystkie elementy są rozmieszczone na stacku, bo teraz część znajduje się w build a część w pomniejszych metodach, a druga sprawa, że samo Positioned wynika z użycia Stack, więc fajnie jakby były na tym samym poziomie abstrakcji :)